index.html

```diff
@@ -80,13 +84,16 @@ <h6 class="tag-name4">html</h6>
             </div>
 
             <div class="actions">
-              <div class="btn">
-                <a href="#">
-                  <h6 class="btn-text">See Project</h6>
-                </a>
+              <div>
+                <button class="project-1 btn" data-modal-target="#modal" type="button"><span class="btn-text">See
+                    Project</span></button>
               </div>
+              
             </div>
+           
           </div>
+          
+          
         </div>
         <div class="card-enable2">
           <div class="img-placeholder2">
```

Comment:
- [ ] [OPTIONAL] Please load the content of the projects section from the javascript array. It is not supposed to be loaded statically in the HTML.